src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/scripted_fields_table/scripted_fields_table.js

```diff
@@ -109,10 +109,11 @@ uiModules.get('apps/management')
 
         $scope.remove = function (field) {
           const confirmModalOptions = {
-            confirmButtonText: 'Delete field',
-            onConfirm: () => { $scope.indexPattern.removeScriptedField(field.name); }
+            confirmButtonText: 'Delete',
+            onConfirm: () => { $scope.indexPattern.removeScriptedField(field.name); },
+            title: `Delete scripted field ${field.name}?`
```

Comment:
@gchaps Should we wrap custom names of things in quotation marks? For example, if the scripted field happens to be named "field", this message will read:

> Delete scripted field field?

It seems like any generic-sounding word will make that sentence read funny. I think quotation marks would clarify the question:

> Delete scripted field "field"?